opensearch_py_ml/dataframe.py

```diff
@@ -424,9 +424,14 @@ def drop(
             axis = pd.DataFrame._get_axis_name(axis)
             axes = {axis: labels}
         elif index is not None or columns is not None:
-            axes, _ = pd.DataFrame()._construct_axes_from_arguments(
-                (index, columns), {}
-            )
+            axes = {
+                "index": to_list_if_needed(index),
+                "columns": (
+                    pd.Index(to_list_if_needed(columns))
+                    if columns is not None
+                    else None
```

Comment:
I noticed that in the implementation of 

opensearch_py_ml.utils that when the value is None that you would return None already maybe we dont need a ternary operation here since its already doing that?

Reply:
Hey Brian,

We can’t remove the ternary operation at this point because calling `pd.Index(None)` results in a `TypeError`. Additionally, we need to keep [this check](https://github.com/Yerzhaisang/opensearch-py-ml/blob/c55639d022ddbdc4732291e6726b3b2c6a5a9bc1/opensearch_py_ml/utils.py#L81) since it can be required [here](https://github.com/Yerzhaisang/opensearch-py-ml/blob/c55639d022ddbdc4732291e6726b3b2c6a5a9bc1/opensearch_py_ml/dataframe.py#L428).